docs/whats_new.rst

```diff
@@ -942,7 +942,7 @@ New Features
     - :attr:`MessageFlags.loading`
     - :attr:`UserFlags.http_interactions_bot`, :attr:`PublicUserFlags.http_interactions_bot`
     - :attr:`UserFlags.spammer`, :attr:`PublicUserFlags.spammer`
-
+    - :attr:`UserFlags.active_application`, :attr:`PublicUserFlags.active_application`
```

Comment:
This does not need to be here, since you have made the changelog entry file already. This is something implemented from #148 and #194, which are old.

Reply:
fixed